docs/reference/migration/migrate_8_0/rollup.asciidoc

```diff
@@ -9,12 +9,18 @@
 
 // end::notable-breaking-changes[]
 
-[float]
-==== StartRollupJob endpoint returns success if job already started
-
+.The StartRollupJob endpoint now returns a success status if a job has already started.
+[%collapsible]
+====
+*Details* +
 Previously, attempting to start an already-started rollup job would
 result in a `500 InternalServerError: Cannot start task for Rollup Job
 [job] because state was [STARTED]` exception.
 
 Now, attempting to start a job that is already started will just
-return a successful `200 OK: started` response.
\ No newline at end of file
+return a successful `200 OK: started` response.
+
+*Impact* +
+Update your workflow and applications to assume a `200` status response for
+requests attempting to start a rollup job that has already started.
```

Comment:
@polyfractal Can you confirm this guidance? If there are other steps users should take, let me know and I'll get it added.

Feel free to ignore the other changes in this PR unless you'd like to review them. These are mostly formatting changes to make the changes collapsible.

Reply:
I might tweak it to say something like:

> Update your workflow and applications to assume that a `200` status in response to attempting to start a rollup job means the job is in an actively started state.  The request itself may have started the job, or it was previously running and so the request had no effect.

Reply:
Thanks @polyfractal. Updated with 9108b404354.

Reply:
@jrodewig LGTM!